CHANGELOG.md

```diff
@@ -3,10 +3,9 @@
 <!--## Unreleased-->
 ## v2.10.1
 
-- Provide a way to update application process by a known process name
+- Add GetChildProcessById, GetAllProcessByName and some methods to fix issue close/quit application

```

Comment:
there is no `UpdateProcessByName` anymore, so let's remove that?
some methods to fix issue close/quit application --> those functions are generally to help working with child process so better --> `some methods to interact with child process`